spec/libsass-closed-issues/issue_1484/error-dart-sass

```diff
@@ -1,6 +1,6 @@
 Error: expected "{".
```

Comment:
The non dart-sass version of the error here seems to be more informative. Should this be expected "}" instead?

Reply:
Yes, probably; this was a pre-existing issue, so I've filed a separate bug to track it: https://github.com/sass/dart-sass/issues/577